backstop_data/engine_scripts/puppet/search/actions/common.js

```diff
@@ -9,8 +9,7 @@
  * @param {CrmPage} page
  * @param {Array} checkboxIds
  */
-module.exports = async (page, checkboxIds = ['3']) => {
-  await page.engine.type('#sort_name', 'Technology');
```

Comment:
why _were there_ organization with "Technology" in the name? What changed?

Reply:
please find this out

Reply:
Added Sort name keyword 'Alliance' to filter out results